src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -138,26 +145,47 @@
          to one of its options).  Simply appending the traits would break code coverage. -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+      <BareTestCommandLine>$(BareTestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</BareTestCommandLine>
+      <OutputFolderForScriptGenerator Condition="'$(TestDisabled)'=='true'">$(OutDir)</OutputFolderForScriptGenerator>
+      <OutputFolderForScriptGenerator Condition="'$(TestDisabled)'!='true'">$(TestPath)%(TestTargetFramework.Folder)</OutputFolderForScriptGenerator>
+      <OutputFolderForScriptGenerator>$(TestPath)%(TestTargetFramework.Folder)</OutputFolderForScriptGenerator>
```

Comment:
Looks like you are always setting this property to the last value here as it doesn't contain any condition. Is that intended?